cache/manager.go

```diff
@@ -321,14 +322,50 @@ func (cm *cacheManager) Prune(ctx context.Context, ch chan client.UsageInfo, opt
 		check = c
 	}
 
-	return cm.prune(ctx, ch, filter, opt.All, check)
+	totalSize := int64(0)
+	if opt.KeepBytes != 0 {
```

Comment:
Sanity checks. It's weird to keep negative bytes, or negative durations.

Reply:
The current behavior should be correct if negative numbers are passed(meaning they behave the same as `0` so this would only be performance and unnecessary for cases that don't appear in practice. In the future, a negative number could mean relative to empty space and not an error at all.